src/components/controls/choose-tip-label.js

```diff
@@ -56,10 +62,7 @@ export function collectAvailableTipLabelOptions(colorings) {
      */
     {value: 'none', label: "none"},
     {value: strainSymbol, label: "Sample Name"},
```

Comment:
One oddity I noticed here is the hardcoding of `node.name` to "Sample Name" which is a little jarring when we use accessions for `node.name`.